packages/manager/apps/public-cloud/src/navbar/navbar-notification.service.js

```diff
@@ -0,0 +1,124 @@
+import _ from 'lodash';
+import moment from 'moment';
+
+export default class NavbarNotificationService {
+  /* @ngInject */
+
+  constructor(
+    $interval,
+    $q,
+    $translate,
+    OvhApiNotificationAapi,
+    TARGET,
+  ) {
+    this.$interval = $interval;
+    this.$q = $q;
+    this.$translate = $translate;
+    this.OvhApiNotificationAapi = OvhApiNotificationAapi;
+    this.TARGET = TARGET;
+
+    this.NOTIFICATION_REFRESH_TIME = 60000;
+  }
+
+  getMessages() {
+    return this.$translate.refresh()
+      .then(() => this.OvhApiNotificationAapi.query({
+        lang: this.$translate.preferredLanguage(),
+        target: this.TARGET,
+      }).$promise
+        .catch((error) => {
```

Comment:
would be better to let the promise be rejected and handle the reject in the caller function